pymc3/variational/updates.py

```diff
@@ -234,7 +234,7 @@ def apply_momentum(updates, params=None, momentum=0.9):
     updates : OrderedDict
         A dictionary mapping parameters to update expressions
     params : iterable of shared variables, optional
-        The variables to apply momentum to. If omitted, will apply
+        The variables to apply momentum too. If omitted, will apply
```

Comment:
Wasn't this originally correct? Shouldn't it be "the variables to apply momentum to"?

Reply:
Gooooooood catch, @eigenfoo! I was not looking closely enough there.

Reply:
I have directly edited the PR. This is fixed now.